public/css/detail/event-rule-detail.less

```diff
@@ -38,6 +30,14 @@
       width: 25em;
     }
   }
+
+  .control-button.disabled {
```

Comment:
There is no hint for delete button, why it is disabled.

Reply:
This has been fixed in the original PR #159